lets-encrypt/test-new-config.sh

```diff
@@ -49,28 +47,29 @@ fi
 # Set trusted domains
 run_script NETWORK trusted
 
-add_crontab_le() {
-# shellcheck disable=SC2016
-DATE='$(date +%Y-%m-%d_%H:%M)'
+# Add crontab
 cat << CRONTAB > "$SCRIPTS/letsencryptrenew.sh"
-#!/bin/sh
-if ! certbot renew --quiet --no-self-upgrade > /var/log/letsencrypt/renew.log 2>&1 ; then
-        echo "Let's Encrypt FAILED!"--$DATE >> /var/log/letsencrypt/cronjob.log
+#!/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
```

Comment:
Without this the renew never works via cronjob in my testing since the snap directory isn't in the cronjobs path variable...